guides/data_and_structure.md

```diff
@@ -0,0 +1,151 @@
+ `publishedAt` | `DateTime` | Time of publishing. Currently also used to determine if the  episode has been published and is public.
+ `number` | `Integer` | "Track" number of the episode in the podcast. aka `episodeNumber`
+ 
+Potential fields open for discussion:  
+
+* `season`
+* `link`
+* More iTunes metadata fields (`itunes:explicit`)
+* Contributors
+ 
+
+### Audio
+
+An **Audio** belongs to at least a **Network** or **Episode** but can be used in multiple locations.
+
+ Field | Type | Description 
+   --: | :--  | :--
+ `title` | `String` | The name of the audio. Mostly used internally or when not used in an episode.
+ `duration` | `Integer` | Duration of the audio microseconds
+ `publishedAt` | `DateTime` | Time of publishing. Currently also used to determine if the  audio has been published and is public.
+ 
+
+Important fields currently missing: 
+
+* `renderedAudioFiles` | `listOf(AudioFiles)` | Audio files representing rendered version of the audio 
+
+
+### AudioFile
+
+An **AudioFile** belongs to an **Audio**
+
+ Field | Type | Description 
+   --: | :--  | :--
+ `title` | `String` | The name of the AudioFile. Maybe this should rather be a filename?
+ `mimeType` | `String` | mime type of the audio file. Usually `audio/mpeg` for `.mp3` or `audio/mp4` for `.m4a`
+ `byteLength` | `Integer` | File size
+ 
+### Chapter
+
+An **Chapter** belongs to an **Audio**
+
+ Field | Type | Description 
+   --: | :--  | :--
+`start` | `Integer` | Start of chapter in microseconds
+`title` | `String` | Title of the chapter
+`image` | `Image` | Chapter image (optional)
+`link`  | `String` | Chapter URL to link to (optional)
+
+### Person
+
+A **Person** belongs to a network. It can take part in contributions.
+
+Field | Type | Description 
+  --: | :--  | :--
+`name` | `String` | The full name of the person                                                                  
+`nick` | `String` | The nickname of the person                                                                   
+`displayName` | `String` | The name that should be used to display this person in public                                
+`link` | `String` | A public accessible http URL representing this person (e.g. social media account or homepage)
+`image` | `Image`  | Avatar image                                                                                 
+
+
+### Contribution
+
+A **Contribution** references a Person and either an **Audio** or a **Podcast**
+
+ Field | Type | Description 
+   --: | :--  | :--
+ `role` | `String` | A contribution role for that person. E.g. `on_air` or `support`
+ 
+ Discussion:
+ 
+ * How much should we formalize this? Currently this is just a text string. I think we should at least make a list of suggestions for the default roles we imagine, that can be offered in frontends and given a good correspondence in the feed metadata too.
```

Comment:
Providing a predefinet set of roles sounds like a worthwhile goal. Reduces complexity and makes the feature more approachable.

Maybe @timpritlove can chime in with his experience of working with Contributor roles & groups in Podlove Publisher.